README.md

```diff
@@ -26,6 +26,8 @@ LevelUP
   * <a href="#contributing">Contributing</a>
   * <a href="#license">Licence &amp; copyright</a>
 
+**Note** If you are upgrading `levelup` to a new major version, please check `CHANGELOG.md` for api changes and upgrade guides.
+
```

Comment:
@vweevers The reason I wrote it this way, is because I wanted to have a generic instruction that will be valid for future major versions.

Reply:
Sure. Could even simplify to:

```markdown
**If you are upgrading:** please see `CHANGELOG.md`.
```